internal/build/path_mapping.go

```diff
@@ -63,7 +65,9 @@ func MountsToPathMappings(mounts []model.Mount) []pathMapping {
 }
 
 // Return all the path mappings for local paths that do not exist.
-func missingLocalPaths(mappings []pathMapping) ([]pathMapping, error) {
+func missingLocalPaths(ctx context.Context, mappings []pathMapping) ([]pathMapping, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "daemon-missingLocalPaths")
```

Comment:
this seems like a weird thing to want spans for? at this point, why not just use a normal prifiler?